<!DOCTYPE html>
<html lang='en'>

<head>
    <base href="..">
    <link rel="shortcut icon" type="image/png" href="assets/logo.png"/>
    <link rel="stylesheet" type="text/css" media="all" href="assets/main.css"/>
    <script type="text/javascript" async
        src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/MathJax.js?config=default">
    </script>
    <meta name="description" content="Conference Template">
    <meta name="resource-type" content="document">
    <meta name="distribution" content="global">
    <meta name="KeyWords" content="Conference">
    <title>Program | SAM 2024</title>
</head>

<body>

    <div class="banner">
        <style>
            img { 
              width: 1000px;
              margin-top: 160px; 
            }
          </style>
        <img src="assets/salzburg.jpg" alt="Conference Template Banner">
        <div class="top-left">
            <!--<span class="title1">Conference</span><span class="title2">Template</span> <span class="year">2525</span>-->
            <span style="color:rgb(0, 85, 255); font-size: 50px">Fifth International Workshop on</span> <br> <br> <span style="color:rgb(0, 85, 255); font-size: 45px"> Statistical Analyses of Multi-Outcome Data</span>
        </div>
        <div class="bottom-right">
            July 9-10, 2024 <br> Salzburg, Austria
        </div>
    </div>

    <table class="navigation">
        <tr>
            <td class="navigation">
                <a title="Conference Home Page" href=".">Home</a>
            </td>
            <td class="navigation">
                <a title="Register for the Conference" href="registration">Registration</a>
            </td>
            <td class="navigation">
                <a title="Conference Submission" href="submission">Submission</a>
            </td>
            <td class="navigation">
                <a class="current" title="Conference Program" href="program">Program</a> 
            </td>
            <td class="navigation">
                <a title="Conference Accommodation" href="accommodation">Accommodation</a>
            </td>
            <td class="navigation">
                <a title="Conference Venue" href="venue">Venue</a>
            </td>
            <td class="navigation">
                <a title="Conference Organizers" href="organizer">Organizers</a>
            </td>
            <td class="navigation">
                <a title="Conference Sponsors" href="sponsor">Sponsors</a>
            </td>
            <td class="navigation">
                <a title="Conference History" href="history">History</a>
            </td>
        </tr>
    </table>

    <h2>Conference Program</h2>

    <table class="styled-table" style="width:70%">
        <thead>
            <tr>
            <th>Time</th>
            <th>7/9/2024</th>
            <th>Time</th>
            <th>7/10/2024</th>
            </tr>
        </thead>
        <tbody>
            <tr>
            <td>8:50-9:00</td>
            <td>Opening Remarks</td>
            <td>9:00-10:45</td>
            <td>Parallel Invited Sessions  </td>
            </tr>
            <tr>
            <td>9:00-10:00</td>
            <td>Keynote 1 by Ian McKeague</td>
            <td>10:45-11:00</td>
            <td>Break </td>
            </tr>
            <tr>
            <td>10:00-10:15</td>
            <td>Break</td>
            <td>11:00-12:45</td>
            <td>Parallel Invited Sessions </td>
            </tr>
            <tr>
                <td>10:15-12:00</td>
                <td>Parallel Invited Sessions </td>
                <td>12:45-1:45</td>
                <td>Lunch</td>
            </tr>
            <tr>
                <td>12:00-1:00</td>
                <td>Lunch</td>
                <td>1:45-3:30 </td>
                <td>Parallel Invited Sessions </td>
            </tr>
            <tr>
                <td>1:00-2:45 </td>
                <td>Parallel Invited Sessions </td>
                <td>3:30-3:45</td>
                <td>Break</td>
            </tr>
            <tr>
                <td>2:45-3:00</td>
                <td>Break</td>
                <td>3:45-4:45 </td>
                <td>Parallel Contributed Sessions</td>
            </tr>
            <tr>
                <td>3:00-4:45 </td>
                <td>Parallel Invited Sessions </td>
                <td>4:45-5:00 </td>
                <td>Break</td>
            </tr>
            <tr>
                <td>4:45-5:00</td>
                <td>Break</td>
                <td>5:00-6:00</td>
                <td>Keynote 2 by Markus Pauly </td>
            </tr>
            <tr>
                <td>5:00-6:00</td>
                <td>Parallel Contributed Sessions </td>
                <td>6:00-6:10</td>
                <td>Closing Remarks</td>
            </tr>
            <tr>
                <td>6:00-8:30</td>
                <td>Banquet/Junior Research Award</td>
                <td></td>
                <td></td>
            </tr>
        </tbody>
    </table>


    <h2>Invited Sessions</h2>

    <button type="button" class="collapsible">Tentative: Extreme value analysis</button>
    <div class="content">
        <table id="PUTSPEAKERNAMEHERE">
            <tr>
                <td class="organizer">
                    Judy Wang (Organizer)
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Daniela Castro-Camilo
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Sebastian Engelke
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Tiandong Wang
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Chen Zhou
                </td>
            </tr>

            <tr>
                <td class="title">
                    A Bayesian multivariate extreme value mixture model
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Daniela Castro-Camilo<br>
                    Lecturer in Statistics, School of Mathematics and Statistics, University of Glasgow, Glasgow G12 8QQ
                    <br>
                    Daniela.CastroCamilo@glasgow.ac.uk
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Impact assessment of natural hazards requires the consideration of both extreme and non-extreme events. Extensive research has been conducted on the joint modelling of bulk and tail in univariate settings; however, the corresponding body of research in the context of multivariate analysis is comparatively scant. This study extends the univariate joint modelling of bulk and tail to the multivariate framework. Specifically, it pertains to cases where multivariate observations exhibit extremity in at least one component.
                    <br>
                    We propose a multivariate extreme value mixture model that assumes a parametric model to capture the bulk of the distribution, which is in the max-domain of attraction of a multivariate extreme value distribution. The multivariate tail is described by the asymptotically justified multivariate generalized Pareto distribution. Bayesian inference based on multivariate random-walk Metropolis-Hastings and the automated factor slice sampler allows us to easily incorporate uncertainty from the threshold selection. The performance of our model is tested using different simulation scenarios, and the applicability of our model is illustrated using temperature records in the UK that show the need to accurately describe the joint tail behaviour.
                </td>
            </tr>

            <tr>
                <td class="title">
                    Machine learning beyond the data range: an extreme value perspective
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Sebastian Engelke<br>
                    Associate Professor, Research Center for Statistics, University of Geneva
                    <br>
                    Daniela.CastroCamilo@glasgow.ac.uk
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Machine learning methods perform well in prediction tasks within the range of the training data. These methods typically break down when interest is in (1) prediction in areas of the predictor space with few or no training observations; or (2) prediction of quantiles of the response that go beyond the observed records. Extreme value theory provides the mathematical foundation for extrapolation beyond the range of the training data, both in the dimension of the predictor space and the response variable. In this talk we present recent methodology that combines this extrapolation theory with flexible machine learning methods to tackle the out-of-distribution generalization problem (1) and the extreme quantile regression problem (2). We show the practical importance of prediction beyond the training observations in environmental and climate applications, where domain shifts in the predictor space occur naturally due to climate change and risk assessment for extreme quantiles is required.
                </td>
            </tr>

            <tr>
                <td class="title">
                    Testing for Strong VS Full Dependence
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Tiandong Wang, Ph.D.<br>
                    Shanghai Center for Mathematical Sciences, Fudan University<br>
                    td_wang@fudan.edu.cn 
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Preferential attachment models of network growth are bivariate heavy tailed models for in- and out-degree with limit measures which either concentrate on a ray of positive slope from the origin or on all of the positive quadrant depending on whether the model includes reciprocity or not. Concentration on the ray is called full dependence. If there were a reliable way to distinguish full dependence from not-full, we would have guidance about which model to choose. This motivates investigating tests that distinguish between (i) full dependence; (ii) strong dependence (limit measure concentrates on a proper subcone of the positive quadrant; (iii) concentration on positive quadrant. We give two test statistics and discuss their asymptotically normal behavior under full and not-full dependence.
                    <br>
                    This is a joint work with Prof. Sidney Resnick at Cornell University. 
                </td>
            </tr>

            <tr>
                <td class="title">
                    Tail copula estimation for heteroscedastic extremes
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Chen Zhou<br>
                    Erasmus University Rotterdam<br>
                    zhou@ese.eur.nl
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Consider independent multivariate random vectors which follow the same copula, but where each marginal distribution is allowed to be non-stationary. This non-stationarity is for each marginal governed by a scedasis function (see Einmahl et al. (2016)) that is the same for all marginals. We establish the asymptotic normality of the usual rank-based estimator of the stable tail dependence function, or, when specialized to bivariate random vectors, the corresponding estimator of the tail copula. Remarkably, the heteroscedastic marginals do not affect the limiting process. Next, under a bivariate setup, we develop nonparametric tests for testing whether the scedasis functions are the same for both marginals. Detailed simulations show the good performance of the estimator for the tail dependence coefficient as well as that of the new tests. In particular, novel asymptotic confidence intervals for the tail dependence coefficient are presented and their good finite-sample behavior is shown. Finally an application to the S&P500 and Dow Jones indices reveals that their scedasis functions are about equal and that they exhibit strong tail dependence. 
                </td>
            </tr>

        </table>
    </div>

    <button type="button" class="collapsible">Tentative: Recent Advances in Neuroimaging Analysis</button>
    <div class="content">
        <table id="PUTSPEAKERNAMEHERE">
            <tr>
                <td class="organizer">
                    Lexin Li (Organizer)
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Hernando Ombao
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Jaroslaw Harezlak
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Haoda Fu
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Lexin Li
                </td>
            </tr>

            <tr>
                <td class="title">
                    Overview of Functional Dependence in Brain Networks
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Hernando Ombao<br>
                    Statistics Program, 
                    King Abdullah University of Science and Technology<br>
                    hernando.ombao@kaust.edu.sa
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Brain activity is complex. A full understanding of brain activity requires careful study of its multi-scale spatial-temporal organization (from neurons to regions of interest; and from transient events to long-term temporal dynamics). Motivated by these challenges, we will explore some characterizations of dependence between components of a brain network. This is potentially interesting because alterations in functional brain connectivity are associated with mental and neurological diseases. In this talk, we provide an overview of functional dependence measures. We present a general framework for exploring dependence through the oscillatory activities derived from each component of the tine series. The talk will draw connections of this framework to some of the classical notions of spectral dependence such as coherence, partial coherence, and dual-frequency coherence. Moreover, this framework provides a starting point for exploring potential non-linear cross-frequency interactions. These interactions include the impact of phase of one oscillatory activity in one component on the amplitude of another oscillation. The proposed approach captures lead-lag relationships and hence can be used as a general framework for spectral causality. Under this framework, we will also present some recent work on inference using spectral mutual information and entropy measures. This is joint work with Marco Pinto (UC Irvine), Paolo Redondo (KAUST) and Raphael Huser (KAUST).
                </td>
            </tr>

            <tr>
                <td class="title">
                    Novel penalized regression method applied to study the association of brain functional connectivity and alcohol drinking
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Jaroslaw (Jarek) Harezlak, Ph.D.<br>
                    Department of Epidemiology and Biostatistics, School of Public Health-Bloomington, 
                    Indiana University, Bloomington, IN<br>
                    harezlak@iu.edu                    
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    The intricate associations between brain functional connectivity and clinical outcomes are difficult to estimate. Common approaches used do not account for the interrelated connectivity patterns in the functional connectivity (FC) matrix, which can jointly and/or synergistically affect the outcomes. In our application of a novel penalized regression approach called SpINNEr (Sparsity Inducing Nuclear Norm Estimator), we identify brain FC patterns that predict drinking outcomes. Results dynamically summarized in the R shiny app indicate that this scalar-on-matrix regression framework via the SpINNEr approach uncovers numerous reproducible FC associations with alcohol consumption. 
                </td>
            </tr>

            <tr>
                <td class="title">
                    LLM Is Not All You Need. Generative AI on Smooth Manifolds 
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Haoda Fu, Ph.D.<br>
                    Associate Vice President, AI/Machine Learning, AADS, Eli Lilly<br>
                    fu_haoda@lilly.com
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Generative AI is a rapidly evolving technology that has garnered significant interest lately. In this presentation, we'll discuss the latest approaches, organizing them within a cohesive framework using stochastic differential equations to understand complex, high-dimensional data distributions. We'll highlight the necessity of studying generative models beyond Euclidean spaces, considering smooth manifolds essential in areas like robotics and medical imagery, and for leveraging symmetries in the de novo design of molecular structures. Our team's recent advancements in this blossoming field, ripe with opportunities for academic and industrial collaborations, will also be showcased.
                </td>
            </tr>

            <tr>
                <td class="title">
                    Kernel Ordinary Differential Equations
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Lexin Li<br>
                    Professor, Department of Biostatistics and Epidemiology & Helen Wills Neuroscience Institute, University of California, Berkeley<br>
                    lexinli@berkeley.edu
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Ordinary differential equation (ODE) is widely used in modeling biological and physical processes in science. In this talk, we propose a new reproducing kernel-based approach for estimation and inference of ODE given noisy observations. We do not assume the functional forms in ODE to be known, or restrict them to be linear or additive, and we allow pairwise interactions. We perform sparse estimation to select individual functionals, and construct confidence intervals for the estimated signal trajectories. We establish the estimation optimality and selection consistency of kernel ODE under both the low-dimensional and high-dimensional settings, where the number of unknown functionals can be smaller or larger than the sample size. Our proposal builds upon the smoothing spline analysis of variance (SS-ANOVA) framework, but tackles several important problems that are not yet fully addressed, and thus extends the scope of existing SS-ANOVA as well. We demonstrate the efficacy of our method through numerous ODE examples.
                </td>
            </tr>

        </table>
    </div>

    <button type="button" class="collapsible">Joint Modeling of Complex Survival Data</button>
    <div class="content">
        <table id="PUTSPEAKERNAMEHERE">
            <tr>
                <td class="organizer">
                    Xinyuan Song (Organizer)
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Xingqiu Zhao
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Liming Xiang
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Jun Ma
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Xinyuan Song
                </td>
            </tr>

            <tr>
                <td class="title">
                    Deep Nonparametric Inference for Conditional Hazard Function
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Xingqiu Zhao<br> 
                    The Hong Kong Polytechnic University<br>
                    xingqiu.zhao@polyu.edu.hk
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    We propose a novel deep learning approach to nonparametric statistical inference for the conditional hazard function of survival time with right-censored data. We use a deep neural network (DNN) to approximate the logarithm of a conditional hazard function given covariates and obtain a DNN likelihood-based estimator of the conditional hazard function. Such an estimation approach grants model flexibility and hence relaxes structural and functional assumptions on conditional hazard or survival functions. We establish the consistency, convergence rate, and functional asymptotic normality of the proposed estimator. Subsequently, we develop new one-sample tests for goodness-of-fit evaluation and two-sample tests for treatment comparison. Both simulation studies and real application analysis show superior performances of the proposed estimators and tests in comparison with existing methods. 
                </td>
            </tr>

            <tr>
                <td class="title">
                    Multiple Imputation for Flexible Modelling of Interval-censored Data with Covariates Subject to Missingness and Detection Limits 
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Liming Xiang<br>
                    Nanyang Tech University<br>
                    LMXiang@ntu.edu.sg
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Interval-censored failure time data is popular in biomedical studies when a failure time is not observed exactly but only known to lie in an interval obtained from a sequence of examination times. The presence of covariates subject to missingness and detection limits poses challenges for regression analysis of interval-censored data and necessitates an eﬀective statistical method. We propose a novel multiple imputation approach via rejection sampling for analysis of such data under semiparametric transformation models. Our proposal alleviates strong dependence of the usual imputation methods on the choice of imputation models and yields consistently and asymptotically normal estimators of the regression parameters. Simulation studies demonstrate that the proposed approach is flexible and leads to more efficient estimation than the complete analysis and augmented inverse probability weighting analysis in various practical situations. Finally, we apply the proposed approach to an Alzheimer’s disease data set that motivates this study. 
                </td>
            </tr>

            <tr>
                <td class="title">
                    Joint modelling of longitudinal covariates and partly-interval censored survival data - a penalized likelihood approach
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Jun Ma<br>
                    Macquarie University<br>
                    jun.ma@mq.edu.au
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    This talk will focus on a joint modelling of longitudinal covariates and partly interval censored time-to-event data. Longitudinal time-varying covariates play a crucial role in achieving accurate dynamic predictions using a survival regression model. However, these covariates are often measured at limited time points and may contain measurement errors. Moreover, they are usually specific to each individual. On the other hand, the event times of interest are often interval-censored. Accounting for all these factors is essential when constructing a survival model. We will present a new approach for joint modelling of the longitudinal time-varying covariates and the time-to-event Cox model, where the latter is subject to interval censoring. We will develop a novel maximum penalized likelihood approach for estimation of all the model parameters including the random effects. A profile likelihood is used to obtain the covariance matrix of the estimated parameters.    
                </td>
            </tr>

            <tr>
                <td class="title">
                    Bayesian tree-based heterogeneous mediation analysis with a time-to-event outcome
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Xinyuan Song<br>
                    Chinese University of Hong Kong<br>
                    xysong@sta.cuhk.edu.hk
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Mediation analysis aims at quantifying and explaining the underlying causal mechanism between an exposure and an outcome of interest. In the context of survival analysis, mediation models have been widely used to achieve causal interpretation for the direct and indirect effects on the survival of interest. Although heterogeneity in treatment effect is drawing increasing attention in biomedical studies, none of the existing methods have accommodated the presence of heterogeneous causal pathways pointing to a time-to-event outcome. In this study, we consider a heterogeneous mediation analysis for survival data based on a Bayesian tree-based Cox proportional hazards model with shared topologies. Under the potential outcomes framework, individual-specific conditional direct and indirect effects are derived on the scale of the logarithm of hazards, survival probability, and restricted mean survival time. A Bayesian approach with efficient sampling strategies is developed to estimate the conditional causal effects through the Monte Carlo implementation of the mediation formula. Simulation studies show the satisfactory performance of the proposed method. The proposed model is then applied to an HIV dataset extracted from the ACTG175 study to demonstrate its usage in detecting heterogeneous causal pathways.
                </td>
            </tr>

        </table>
    </div>
    

    <button type="button" class="collapsible">Analysis of data with multiple treatments and mixed outcomes</button>
    <div class="content">
        <table id="PUTSPEAKERNAMEHERE">
            <tr>
                <td class="organizer">
                    Lan Luo (Organizer)
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Emily Hector
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Lan Luo
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Ling Zhou
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Liangyuan Hu
                </td>
            </tr>

            <tr>
                <td class="title">
                    Turning the data-integration dial: efficient inference from different data sources
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Emily Hector<br>
                    University of North Carolina, Chapel Hill
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    A fundamental aspect of statistics is the integration of data from different sources. Classically, Fisher and others were focused on how to integrate homogeneous sets of data. More recently, the question of if data sets from different sources should be integrated is becoming more relevant. The current literature treats this as a yes/no question: integrate or don't. Here we take a different approach, motivated by information-sharing principles coming from the shrinkage estimation literature. In particular, we deviate from the binary, yes/no perspective and propose a dial parameter that controls the extent to which two data sources are integrated. How far this dial parameter should be turned is shown to depend on the informativeness of the different data sources as measured by Fisher information. This more-nuanced data integration framework leads to relatively simple parameter estimates and valid tests/confidence intervals. We demonstrate both theoretically and empirically that setting the dial parameter according to our recommendation leads to more efficient estimation compared to other binary data integration schemes.
                </td>
            </tr>

            <tr>
                <td class="title">
                    Efficient quantile covariate adjusted response adaptive experiments
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Lan Luo<br>
                    Assistant Professor, Department of Biostatistics and Epidemiology, Rutgers School of Public Health<br>
                    ll1118@sph.rutgers.edu
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    In program evaluation studies, understanding the heterogeneous distributional impacts of a program beyond the average effect is crucial. Quantile treatment effect (QTE) provides a natural measure to capture such heterogeneity. While much of the existing work for estimating QTE has focused on analyzing observational data based on untestable causal assumptions, little work has gone into designing randomized experiments specifically for estimating QTE. In this talk, we propose two covariate adjusted response adaptive design strategies–fully adaptive designs and multi-stage designs–to efficiently estimate the QTE. We demonstrate that the QTE estimator obtained from our designs attains the optimal variance lower bound from a semiparametric theory perspective, which does not impose any parametric assumptions on underlying data distributions. Moreover, we show that using continuous covariates in multi-stage designs can improve the precision of the estimated QTE compared to the classical fully adaptive setting. We illustrate the finite-sample performance of our designs through Monte Carlo experiments and one synthetic case study on charitable giving. Our proposed designs offer a new approach to conducting randomized experiments to estimate QTE, which can have important implications for policy and program evaluation.
                </td>
            </tr>

            <tr>
                <td class="title">
                    High-dimensional subgroup learning for multiple mixed outcome 
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Ling Zhou<br>
                    Southwestern University of Finance and Economics<br>
                    zhouling@swufe.edu.cn
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    In survey research, it is interesting to infer the grouped association patterns between risk factors and questionnaire responses, where the grouping is shared across multiple response variables that jointly capture one’s underlying status. In particular, based on a survey study named the China Health and Retirement Survey (CHRS), our aim is to identify the important risk factors that are simultaneously associated with the health and well-being of senior adults. While earlier studies have pointed to several known risk factors, heterogeneity in the outcome-risk factor association exists, motivating us to analyze this data through the lens of subgroup analysis. We devise a subgroup analysis procedure that models a multiple mixed outcome which describe one’s general health and well-being, while tackling additional challenges that have arisen in our data analysis, including high-dimensionality, collinearity, and weak signals in covariates. Computationally, we propose an efficient algorithm that alternately updates a set of estimating equations and likelihood functions. Theoretically, we establish the asymptotic consistency and normality of the proposed estimators. The validity of our proposal is corroborated by simulation experiments. An application of the proposed method to the CHRS data identifies caring for grandchildren as a new risk factor for poor physical and mental health.       
                </td>
            </tr>

            <tr>
                <td class="title">
                    Estimating the causal effect of multiple intermittent treatments on censored survival outcomes
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Liangyuan Hu<br>
                    Rutgers University<br>
                    lh707@sph.rutgers.edu
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    To draw real-world evidence about the comparative effectiveness of multiple time-varying treatments on patient survival, we develop a joint marginal structural survival model and a novel weighting strategy to account for time-varying confounding and censoring. Our methods formulate complex longitudinal treatments with multiple start/stop switches as the recurrent events with discontinuous intervals of treatment eligibility. We derive the weights in continuous time to handle a complex longitudinal dataset without the need to discretize or artificially align the measurement times. We further use machine learning models designed for censored survival data with time-varying covariates and the kernel function estimator of the baseline intensity to efficiently estimate the continuous-time weights. Our simulations demonstrate that the proposed methods provide better bias reduction and nominal coverage probability when analyzing observational longitudinal survival data with irregularly spaced time intervals, compared to conventional methods that require aligned measurement time points. We apply the proposed methods to a large-scale COVID-19 dataset to estimate the causal effects of several COVID-19 treatments on the composite of in-hospital mortality and ICU admission.
                </td>
            </tr>
        </table>
    </div>
    
    <button type="button" class="collapsible">TBD</button>
    <div class="content">
        <table id="PUTSPEAKERNAMEHERE">
            <tr>
                <td class="organizer">
                    Ying Wei (Organizer)
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Shuang Wang
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Rui Duan
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Yanyuan Ma
                </td>
            </tr>
            <tr>
                <td class="speaker">
                    Ying Wei
                </td>
            </tr>

            <tr>
                <td class="title">
                    PartIES: a disease subtyping framework with Partition-level Integration using diffusion-Enhanced Similarities from Multi-omics Data
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Yuqi Miao<sup>1</sup>, Huang Xu<sup>2</sup>, Shuang Wang<sup>1*</sup><br>
                    1. Department of Biostatistics, Columbia University, New York New York USA<br>
                    2. Department of Statistics, University of Science and Technology of China, Hefei, Anhui, P.R. China<br>
                    sw2206@cumc.columbia.edu
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Integrating multi-omics data helps identify disease subtypes. Many similarity-based clustering methods were developed for disease subtyping using multi-omics data, with many of them focusing on extracting common clustering structures across multiple types of omics data, thus are not meant to preserve specific clustering structures within each omics data type. Moreover, clustering performance of similarity-based methods are known to be affected by how accurate similarity measures are. In this paper, we proposed PartIES, a Partition-level Integration using diffusion-Enhanced Similarities (PartIES) to perform disease subtyping using multi-omics data. We propose to use diffusion to reduce noises in similarity/kernel matrices, and partition individual diffusion-enhanced similarity matrices before integration and learn the integrative similarity structure adaptively on the partition level. Simulation studies showed that the diffusion step enhances clustering accuracy, and PartIES outperforms other competing methods, especially when omics data types provide different clustering structures. Using mRNA, lncRNA, miRNA expression data, DNA methylation data and mutation data from The Cancer Genome Atlas project (TCGA), PartIES identified subtypes in bladder urothelial carcinoma (BLCA) and thyroid carcinoma (THCA) that most significantly differentiate patient survival among all methods. To provide the biological meaning of the identified cancer subtypes, we further mapped subtype-differentiated omics features to the protein-protein interaction (PPI) network to identify important interacting cancer genes and compared the activities of cancer-related pathways across subtypes.
                </td>
            </tr>

            <tr>
                <td class="title">
                    Robust angle-based transfer learning in high dimensions
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Rui Duan, PhD<br>
                    Assistant Professor of Biostatistics, Harvard T.H. Chan School of Public Health<br>
                    Email: rduan@hsph.harvard.edu
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Transfer learning aims to improve the performance of a target model by leveraging data from related source populations, which is known to be especially helpful in cases with insufficient target data. In this paper, we study the problem of how to train a high-dimensional ridge regression model using limited target data and existing regression models trained in heterogeneous source populations. We consider a practical setting where only the parameter estimates of the fitted source models are accessible, instead of the individual-level source data. Under the setting with only one source model, we propose a novel flexible angle-based transfer learning (angleTL) method, which leverages the concordance between the source and the target model parameters. We show that angleTL unifies several benchmark methods by construction, including the target-only model trained using target data alone, the source model fitted on source data, and distance-based transfer learning method that incorporates the source parameter estimates and the target data under a distance-based similarity constraint. We also provide algorithms to effectively incorporate multiple source models accounting for the fact that some source models may be more helpful than others. Our high-dimensional asymptotic analysis provides interpretations and insights regarding when a source model can be helpful to the target model, and demonstrates the superiority of angleTL over other benchmark methods. We perform extensive simulation studies to validate our theoretical conclusions and show the feasibility of applying angleTL to transfer existing genetic risk prediction models across multiple biobanks.
                </td>
            </tr>

            <tr>
                <td class="title">
                    Doubly Flexible Estimation under Label Shift
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Yanyuan Ma<br>
                    Pennsylvania State University<br>
                    yanyuanma@gmail.com
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    In studies ranging from clinical medicine to policy research, complete data are usually available from a population P, but the quantity of interest is often sought for a related but different population Q which only has partial data. In this paper, we consider the setting that both outcome Y and covariate X are available from P whereas only X is available from Q, under the so-called label shift assumption, i.e., the conditional distribution of X given Y remains the same across the two populations. To estimate the parameter of interest in population Q via leveraging the information from population P, the following three ingredients are essential: (a) the common conditional distribution of X given Y, (b) the regression model of Y given X in population P, and (c) the density ratio of the outcome Y between the two populations. We propose an estimation procedure that only needs some standard nonparametric regression technique to approximate the conditional expectations with respect to (a), while by no means needs an estimate or model for (b) or (c); i.e., doubly flexible to the possible model misspecifications of both (b) and (c). This is conceptually different from the well-known doubly robust estimation in that, double robustness allows at most one model to be misspecified whereas our proposal here can allow both (b) and (c) to be misspecified. This is of particular interest in our setting because estimating (c) is difficult, if not impossible, by virtue of the absence of the Y-data in population Q. Furthermore, even though the estimation of (b) is sometimes off-the-shelf, it can face curse of dimensionality or   computational challenges. We develop the large sample theory for the proposed estimator, and examine its finite-sample performance through simulation studies as well as an application to the MIMIC-III database.
                </td>
            </tr>

            <tr>
                <td class="title">
                    A Double Projection Approach for Safe and Efficient Semi-Supervised Data-Fusion
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Ying Wei<br>
                    Columbia University<br>
                    yw2148@cumc.columbia.edu
                </td>
            </tr>
            <tr>
                <td class="abstract">
                    Advances in data collection and transmission technologies have made larger amounts of data readily available. However, there are differences in the data collection capabilities of different data centers, or there are inevitable data missing. Many previous approaches to handling missing information have solely focused on either missing predictors or missing responses. In this paper, we will consider both types of missing and incorporate more information by projecting score functions into subsets, thus proposing algorithms that have ensured efficiency relative to the complete-case analysis. By generalizing the algorithm of this paper, it is promising to be able to handle more complex missing data structures in the future. This is joint work with Molei Liu, Yiming Li and Sean Yang. 
                </td>
            </tr>

        </table>
    </div>
    

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Analysis of multi-dimensional correlated data
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Peter Song (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Annie Qu
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Heping Zhang
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Ji Zhu
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Jian Kang
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Statistical Methods for Metagenomic Data
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Gen Li (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Hongzhe Li
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Huilin Li
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Zhigang Li
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Gen Li
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                TBD
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Shuangge Ma (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Hao Mei
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Yuan Huang
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Shuangge Ma
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Ai-Ling Hour
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Tentative: Advance Development and Application of Joint modeling
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Cheng Zheng (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Ying Zhang
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Cheng Zheng
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Ping Ma
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Danping Liu
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Tentative: Statistical and machine learning methods for data integration in biomedical research
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Qi Long (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Ying Guo
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Suprateek Kundu
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Ming Wang
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Ziyi Li
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Tentative: Generalizability consideration in heterogeneous data
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Menggang Yu (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Jeremy Taylor
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Lu Tian
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Menggang Yu
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Ruth Pfeiffer
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Tentative: Survival analysis and its applications
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Yichuan Zhao (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Gang Li
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Catherine Liu
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Yi Li
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Yichuan Zhao
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Analysis of multi-outcomes and multi-source data
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Zhezhen Jin (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Xiaonan Xue
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Yongzhao Shao
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Shanshan Ding
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Lihui Zhao
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Recent Development of Statistical Methods for Complex Survival Data in Medical Studies
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Yanqing Sun (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Wenqing He
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Yanqing Sun
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Chiung-Yu Huang
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Grace Yi
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Recent developments in Survival Analysis and Statistical Machine Learning
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Tony Sun (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Jian Huang
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Chengchun Shi
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Yifan Cui
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Qixian Zhong
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Flexible statistical learning for complex data
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Yufeng Liu (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Eric Chi
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Ali Shojaie
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Zhengyuan Zhu
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Yufeng Liu
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                The integration of high dimensional data in JOiNT MODELS
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Virginie Rondeau (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Pedro Miranda AFONSO
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Cecile Proust Lima
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Denis Rustand
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Manel Rakez
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Dynamic prediction
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Jessica Barrett (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Hein Putter
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Dimitris Rizopoulos
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Danilo Alvares
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Liang Li
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Dependence Modeling
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Wolfgang Trutschnig & Sebastian Fuchs (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Damjana Kokol Bukovsek
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Nik Stopar
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Jonathan Ansari
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Patrick Langthaler
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Multivariate and Complex Longitudinal Data
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Georg Zimmermann (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Geert Molenberghs
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Frank Konietschke
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Somnath Datta
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Kelly van Lancker
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Precision Medicine and Reinforcement Learning
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Simon Hirländer (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Danyu Lin
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Richard Cook
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Limin Peng
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Lei Liu
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                General topics in multi-outcome data
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Zhezhen Jin (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Xuewen Lu
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Yingwei Paul Peng
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Yuping Wang
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Nicolas Dietrich
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                TBD
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Gang Li (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Hua Zhou
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Lang Wu
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Ying Lu
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Jin Zhou
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="title">
                Topics in Advanced Multi-outcome Data Analysis
            </td>
        </tr>
        <tr>
            <td class="organizer">
                Lei Liu (Organizer)
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Jianwen Cai
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Donglin Zeng
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Yuanjia Wang
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Haibo Zhou
            </td>
        </tr>
    </table>

    <script>
        var coll = document.getElementsByClassName("collapsible");
        var i;
        
        for (i = 0; i < coll.length; i++) {
          coll[i].addEventListener("click", function() {
            this.classList.toggle("active");
            var content = this.nextElementSibling;
            if (content.style.display === "block") {
              content.style.display = "none";
            } else {
              content.style.display = "block";
            }
          });
        }
    </script>
<!--
    <table>
        <tr>
            <td class="date" rowspan="2">
                8:30am
            </td>
            <td class="title-special">
                Registration and Coffee &amp; Tea!
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Here&rsquo;s a section in the program that isn&rsquo;t a talk. 
                Notice that the title is styled differently than the ones for talks.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                9:00am
            </td>
            <td class="title">
                Interesting Title of a Talk 
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Speaker Name (University)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Here&rsquo;s where you put the abstract of the talk. Notice that with MathJax you can include mathematical notation in the abstract like \(f(x)=3x-7\) or like \(h^0(X_t,\omega_{X_t}^{\otimes m})\) or whatever you want. Just notice the MathJax line in the head of this HTML document that allows this. I&rsquo;m going to blab just a tiny bit more in this abstract, but all the other abstracts below, except the ones for the plenary talks, will be just <i>Lorem Ipsum</i> text that needs to be replaced by real abstracts.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                9:25am
            </td>
            <td class="title">
                Interesting Title of a Talk 
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Speaker Name (University)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Filler text: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum dignissim et est et euismod. Fusce et metus tempus, pellentesque ex at, convallis nulla. Ut fringilla commodo tincidunt. Fusce sed est eu massa placerat iaculis eu at mauris. Nullam ut mollis nisi, quis malesuada risus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam ipsum tortor, suscipit non tincidunt vel, bibendum in libero. Nulla facilisi. Pellentesque vitae neque metus. Cras quis est pharetra, vestibulum nisl et, viverra ipsum. Etiam porta dignissim purus, quis tempor metus volutpat eu. Praesent pulvinar libero eget purus tincidunt finibus.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                9:50am
            </td>
            <td class="title">
                Interesting Title of a Talk 
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Speaker Name (University)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Filler text: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum dignissim et est et euismod. Fusce et metus tempus, pellentesque ex at, convallis nulla. Ut fringilla commodo tincidunt. Fusce sed est eu massa placerat iaculis eu at mauris. Nullam ut mollis nisi, quis malesuada risus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam ipsum tortor, suscipit non tincidunt vel, bibendum in libero. Nulla facilisi. Pellentesque vitae neque metus. Cras quis est pharetra, vestibulum nisl et, viverra ipsum. Etiam porta dignissim purus, quis tempor metus volutpat eu. Praesent pulvinar libero eget purus tincidunt finibus.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                10:15am
            </td>
            <td class="title">
                Interesting Title of a Talk 
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Speaker Name (University)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Filler text: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum dignissim et est et euismod. Fusce et metus tempus, pellentesque ex at, convallis nulla. Ut fringilla commodo tincidunt. Fusce sed est eu massa placerat iaculis eu at mauris. Nullam ut mollis nisi, quis malesuada risus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam ipsum tortor, suscipit non tincidunt vel, bibendum in libero. Nulla facilisi. Pellentesque vitae neque metus. Cras quis est pharetra, vestibulum nisl et, viverra ipsum. Etiam porta dignissim purus, quis tempor metus volutpat eu. Praesent pulvinar libero eget purus tincidunt finibus.
            </td>
        </tr>
    </table>

    <table>
        <tr>
            <td class="date" rowspan="2">
                10:40am
            </td>
            <td class="title-special">
                Coffee &amp; Tea Break
            </td>
        </tr>
        <tr>
            <td class="abstract">
                
            </td>
        </tr>
    </table>

    <table class="plenary">
        <tr>
            <td class="date" rowspan="3">
                11:00am
            </td>
            <td class="title">
                This is a Plenary Talk!
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Plenary Speaker
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Here&rsquo;s the abstract for the plenary talk! Notice again that the formatting of this time-block is a bit different that the rest of of the talks.
                Filler text: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum dignissim et est et euismod. Fusce et metus tempus, pellentesque ex at, convallis nulla. Ut fringilla commodo tincidunt. Fusce sed est eu massa placerat iaculis eu at mauris. Nullam ut mollis nisi, quis malesuada risus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam ipsum tortor, suscipit non tincidunt vel, bibendum in libero. Nulla facilisi. Pellentesque vitae neque metus. Cras quis est pharetra, vestibulum nisl et, viverra ipsum. Etiam porta dignissim purus, quis tempor metus volutpat eu. Praesent pulvinar libero eget purus tincidunt finibus.
            </td>
        </tr>
    </table>

    <table>
        <tr>
            <td class="date" rowspan="2">
                Noon
            </td>
            <td class="title-special">
                Lunch
            </td>
        </tr>
        <tr>
            <td class="abstract">
                
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                1:30pm
            </td>
            <td class="title">
                Interesting Title of a Talk 
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Speaker Name (University)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Filler text: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum dignissim et est et euismod. Fusce et metus tempus, pellentesque ex at, convallis nulla. Ut fringilla commodo tincidunt. Fusce sed est eu massa placerat iaculis eu at mauris. Nullam ut mollis nisi, quis malesuada risus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam ipsum tortor, suscipit non tincidunt vel, bibendum in libero. Nulla facilisi. Pellentesque vitae neque metus. Cras quis est pharetra, vestibulum nisl et, viverra ipsum. Etiam porta dignissim purus, quis tempor metus volutpat eu. Praesent pulvinar libero eget purus tincidunt finibus.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                1:55pm
            </td>
            <td class="title">
                Interesting Title of a Talk 
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Speaker Name (University)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Filler text: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum dignissim et est et euismod. Fusce et metus tempus, pellentesque ex at, convallis nulla. Ut fringilla commodo tincidunt. Fusce sed est eu massa placerat iaculis eu at mauris. Nullam ut mollis nisi, quis malesuada risus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam ipsum tortor, suscipit non tincidunt vel, bibendum in libero. Nulla facilisi. Pellentesque vitae neque metus. Cras quis est pharetra, vestibulum nisl et, viverra ipsum. Etiam porta dignissim purus, quis tempor metus volutpat eu. Praesent pulvinar libero eget purus tincidunt finibus.
            </td>
        </tr>
    </table>

    <table>
        <tr>
            <td class="date" rowspan="2">
                2:50pm
            </td>
            <td class="title-special">
                Coffee &amp; Tea Break
            </td>
        </tr>
        <tr>
            <td class="abstract">
                 
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                3:10pm
            </td>
            <td class="title">
                Interesting Title of a Talk 
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Speaker Name (University)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Filler text: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum dignissim et est et euismod. Fusce et metus tempus, pellentesque ex at, convallis nulla. Ut fringilla commodo tincidunt. Fusce sed est eu massa placerat iaculis eu at mauris. Nullam ut mollis nisi, quis malesuada risus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam ipsum tortor, suscipit non tincidunt vel, bibendum in libero. Nulla facilisi. Pellentesque vitae neque metus. Cras quis est pharetra, vestibulum nisl et, viverra ipsum. Etiam porta dignissim purus, quis tempor metus volutpat eu. Praesent pulvinar libero eget purus tincidunt finibus.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                3:35pm
            </td>
            <td class="title">
                Interesting Title of a Talk 
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Speaker Name (University)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Filler text: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum dignissim et est et euismod. Fusce et metus tempus, pellentesque ex at, convallis nulla. Ut fringilla commodo tincidunt. Fusce sed est eu massa placerat iaculis eu at mauris. Nullam ut mollis nisi, quis malesuada risus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam ipsum tortor, suscipit non tincidunt vel, bibendum in libero. Nulla facilisi. Pellentesque vitae neque metus. Cras quis est pharetra, vestibulum nisl et, viverra ipsum. Etiam porta dignissim purus, quis tempor metus volutpat eu. Praesent pulvinar libero eget purus tincidunt finibus.
            </td>
        </tr>
    </table>

    <table class="plenary">
        <tr>
            <td class="date" rowspan="3">
                4:00pm
            </td>
            <td class="title">
                This is Another Plenary Talk!
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Plenary Speaker
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Filler text: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum dignissim et est et euismod. Fusce et metus tempus, pellentesque ex at, convallis nulla. Ut fringilla commodo tincidunt. Fusce sed est eu massa placerat iaculis eu at mauris. Nullam ut mollis nisi, quis malesuada risus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam ipsum tortor, suscipit non tincidunt vel, bibendum in libero. Nulla facilisi. Pellentesque vitae neque metus. Cras quis est pharetra, vestibulum nisl et, viverra ipsum. Etiam porta dignissim purus, quis tempor metus volutpat eu. Praesent pulvinar libero eget purus tincidunt finibus.
            </td>
        </tr>
    </table>
-->
    <!--<footer>
        &copy; Conference Organizers
        &nbsp;|&nbsp; <a href="https://profiles.wustl.edu/en/persons/lei-liu">Lei Liu</a>
    </footer>-->

</body>
</html>

